### ISSUE TITLE
Unable to create poetry environment with python3.10

### ISSUE THREAD

**Issue by ethanabrooks** [python-poetry/poetry#5431]
- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.

- **OS version and name**:  Ubuntu 20.04.4 LTS
- **Poetry version**: 1.1.13
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://gist.github.com/ethanabrooks/1ec5eada2e26f346b83eb46a4bdebc8b

## Issue
I am unable to create a poetry environment with python3.10.
```
> poetry shell
The currently activated Python version 3.8.10 is not supported by the project (^3.10).
Trying to find and use a compatible version.
Using python3.10 (3.10.4)
Creating virtualenv poetry-issue-wxXJaBmj-py3.10 in /home/ethanbro/.cache/pypoetry/virtualenvs
Virtual environment already activated: /home/ethanbro/.cache/pypoetry/virtualenvs/poetry-issue-wxXJaBmj-py3.10
> which python
/usr/bin/python
> ls /home/ethanbro/.cache/pypoetry/virtualenvs/poetry-issue-wxXJaBmj-py3.10
lib  local  pyvenv.cfg 
```
As you can see, there is no `bin/` directory under `/home/ethanbro/.cache/pypoetry/virtualenvs/poetry-issue-wxXJaBmj-py3.10`.

If I change my `pyproject.toml` to `python3.9` and then run `poetry env use $(which python3.9)`, everything works fine (the virtualenv has a `bin` directory).

**Comment by danielhoherd** [comment]
Same problem. Tested on stock ubuntu 22.04 which comes with python 3.10

```
dho@compuper:~/code/blarg$ python --version --version
Python 3.10.4 (main, Apr  2 2022, 09:04:19) [GCC 11.2.0]
dho@compuper:~/code/blarg$ poetry --version
Poetry version 1.1.13
dho@compuper:~/code/blarg$ cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04 (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
dho@compuper:~/code/blarg$ dpkg -l python3
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version         Architecture Description
+++-==============-===============-============-=========================================================================
ii  python3        3.10.4-0ubuntu2 amd64        interactive high-level object-oriented language (default python3 version)
```

Additionally, I am seeing that there is no /bin/activate

```
dho@compuper:~/code/blarg$ poetry shell
The virtual environment found in /home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10 seems to be broken.
Recreating virtualenv blarg-tow_XpAb-py3.10 in /home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10
Spawning shell within /home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10
dho@compuper:~/code/blarg$ . /home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10/bin/activate
bash: /home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10/bin/activate: No such file or directory
dho@compuper:~/code/blarg$ find /home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10/bin
find: ‘/home/dho/.cache/pypoetry/virtualenvs/blarg-tow_XpAb-py3.10/bin’: No such file or directory
```

This is causing `poetry install` to attempt to modify system files:

```
  • Removing colorama (0.4.4): Failed

  EnvCommandError

  Command ['pip', 'uninstall', 'colorama', '-y'] errored with the following return code 2, and output:
  Found existing installation: colorama 0.4.4
  Uninstalling colorama-0.4.4:
  ERROR: Exception:
  Traceback (most recent call last):
    File "/usr/lib/python3.10/shutil.py", line 813, in move
      os.rename(src, real_dst)
  PermissionError: [Errno 13] Permission denied: '/usr/lib/python3/dist-packages/colorama' -> '/tmp/pip-uninstall-rr7nq65y'
```

**Comment by expliyh** [comment]
I also had the same problem, I created a virtual environment using `poetry env use python3.10` , but when I try `poetry install` , I was prompted `The virtual environment found seems to be broken` , also there is no bin/ in the venv directory. And poetry install the package to `$HOME/.local/lib/python3.10/site-packages` instead of the virtual environment directory.

我也遇到了相同的问题，我使用 `poetry env use python3.10` 创建了一个虚拟环境，但当我使用 `poetry install` 时，我被提示 `The virtual environment found seems to be broken`，在venv目录中同样没有 bin/。并且 poetry将软件包安装到了 `$HOME/.local/lib/python3.10/site-packages` 而不是虚拟环境目录。

**Comment by christian-nils** [comment]
Same issue for me with the "The virtual environment found seems to be broken" message with poetry version 1.2.0b3.dev0 (installed from git@master) and with python 3.10. Works perfectly fine with python 3.9.

On Fedora 36.

**Comment by johnziebro** [comment]
Same issue as above. I've completely uninstalled Poetry and reinstalled and done the same with Pyenv.

**Ubuntu 22.04
Pyenv 2.3.1 at /home/anon/.pyenv/bin/pyenv
Poetry 1.1.13 at /home/anon/.local/bin/poetry**

Pyenv local python version:
`cat .python-version`
> 3.10.5
`pyenv versions`
>   system
> \* 3.10.5 (set by /media/anon/36782A484452D4B2/Projects/airtravel/.python-version)

`poetry install`
> **_The currently activated Python version 3.10.4 is not supported by the project (^3.10.5).
Trying to find and use a compatible version. 
Using python3 (3.10.5)_**

In fact python 3.10.5 is not used and 3.10.4 is:
`cat .venv/pyvenv.cfg`
> home = /usr
> implementation = CPython
> version_info = 3.10.4.final.0
> virtualenv = 20.14.1
> include-system-site-packages = false
> base-prefix = /usr
> base-exec-prefix = /usr
> base-executable = /usr/bin/python3.10

`poetry shell`
`python`
> Python 3.10.4 (main, Apr  2 2022, 09:04:19) [GCC 11.2.0] on linux

`poetry config --list`
> cache-dir = "/home/anon/.cache/pypoetry"
> experimental.new-installer = true
> installer.parallel = true
> virtualenvs.create = true
> virtualenvs.in-project = true
> virtualenvs.path = "{cache-dir}/virtualenvs"  # /home/anon/.cache/pypoetry/virtualenvs

**Comment by johnziebro** [comment]
Issue [5190](https://github.com/python-poetry/poetry/issues/5190) appears to be a similar or duplicate issue and references an issue in the new Poetry install script.

**Comment by johnziebro** [comment]
Poetry [Preview Release 1.2.0b1](https://github.com/python-poetry/poetry/pull/5280) addresses this issue:
> _Fixed an issue where the reported python version used for venv creation wasn't correct (https://github.com/python-poetry/poetry/pull/5086)._

Steps to get a working environment:
* Uninstalled Poetry
* Installed the preview release 1.2.0b1:
`curl -sSL https://install.python-poetry.org | python3 - --preview`
* Explicitly define the env's python version:
`poetry env use 3.10.5`

Using `poetry shell` or `poetry install` still encounters the issue of installing the system python version 3.10.4 and ignores 3.10.5 defined in .python-version. The fix is to define the env explicitly as detailed above before issuing 'install'.

TLDR: Install Poetry preview version and define the env python version explicitly with `poetry env use PYTHON_VERSION` substituting your pyenv installed version of python for PYTHON_VERSION.

**Comment by ethanabrooks** [comment]
I tried uninstalling and reinstalling, but still encountering the same issue. I am not using pyenv, for the record.

**Comment by nirvana-msu** [comment]
The new `prefer-active-python` setting with a `false` default was such a gotcha for someone who's been relying on `pyenv` all along... It should be made a lot more prominent in the docs.

**Comment by randolf-scholz** [comment]
For me it worked again after purging poetry config files and reinstalling.
